### PR TITLE
Update QUEST.tsv [Lines 1651-1800]

### PR DESCRIPTION
**Pull request by ghost**
Lines 1651-1800 reviewed. Lines that disappeared after the merge to CBT re-added. Open to feedback.

Again, changed "contaminating gap" to "corruption breach." If anyone can think of a better keyword or think the original is fine then I'm okay with changing it to that or back to the original.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -1649,151 +1649,154 @@ QUEST_20150323_001648	Huh?
...
+QUEST_20150323_001715	$Old Seeds
+QUEST_20150323_001716	$Shaton's Stubbornness
 QUEST_20150401_001717	Star Statue
 QUEST_20150401_001718	Nothing is happening…
-QUEST_20150401_001719	Supressing Altar
-QUEST_20150401_001720	Altar is already activated
-QUEST_20150401_001721	The candlestick is already lit
-QUEST_20150401_001722	Need something to light it…
+QUEST_20150401_001719	$Suppressing Altar
+QUEST_20150401_001720	$The altar is already activated.
```
Comment (Sunaries) [inline QUEST.tsv:1720]: The -> This altar? (since there are many altars in the world of ToS)

Comment (ghost) [inline QUEST.tsv:1720]: I assume the dialogue appears after interacting with the suppressing altar so it should make sense. Not sure.

Comment (grillo-delmal) [inline QUEST.tsv:1720]: If it's a dialogue between 2 characters, "this" would fit, but it seems to be an interaction with the altar, so "The altar" would fit in that context.

**Comment by ghost** [comment]
I guess this is good to go.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**